File: Divyansh Singh/Array/widest vertical area between two points containing no points.cpp
class Solution {
public:
    int maxWidthOfVerticalArea(vector<vector<int>>& points) {
        vector<int>x;
        for(int i=0;i<points.size();i++){
            x.push_back(points[i][0]);
        }
        sort(x.begin(),x.end());
        int maxi=-1;
        for(int i=1;i<x.size();i++){
            maxi=max(maxi,x[i]-x[i-1]);
        }
        return maxi;
        
    }
};

File: Khushboo Singh/Array/Widest Vertical Area Between Two Points Containing No Points.java
class Solution {
    public int maxWidthOfVerticalArea(int[][] points) {
        int[] arr=new int[points.length];
        int max=0;
        for(int i=0;i<points.length;i++){
            arr[i]=points[i][0];
        }
        Arrays.sort(arr);
        for(int i=0;i<arr.length-1;i++){
            if(arr[i+1]-arr[i]>max){
            max=arr[i+1]-arr[i];
            }
            
        }
        return max;
       
    }
}
